buildenv/jenkins/JenkinsfileBase

```diff
@@ -685,18 +685,26 @@ def post(output_name) {
 			}
 
 			junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/work/**/*.jtr.xml, **/junitreports/**/*.xml, **/external_test_reports/**/*.xml'
-			archiveSHAFile()
-			archiveTestenvFile()
-			archiveAQACertFile()
+
+
+            //Defining the files to be archived
+            def testenvFile = "aqa-tests/testenv/testenv.properties"
+            def shaFile = "aqa-tests/TKG/SHA.txt"
+            def AQACertFile = "aqa-tests/TKG/AQACert.log"
+
+            //call the archive function for each file
+            archiveFile(testenvFile)
+	    archiveFile(shaFile)
+            archiveFile(AQACertFile)
```

Comment:
As those filename is only used once may not need to define the variable. 

```
            archiveFile("aqa-tests/testenv/testenv.properties")
	    archiveFile("aqa-tests/TKG/SHA.txt")
            archiveFile("aqa-tests/TKG/AQACert.log")
```

Reply:
@Stergios13 - can you update your PR based on @sophia-guo's suggestion (pass the filenames directly, as we do not need to separately define variables for our purpose.

Reply:
Sorry for being inactive i was busy due to university purposes. Of course i will during the day!